src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -658,5 +658,17 @@ internal static string GetMemberCallerName(this Symbol member)
                 member.IsExplicitInterfaceImplementation() ? ExplicitInterfaceHelpers.GetMemberNameWithoutInterfaceName(member.Name) :
                 member.Name;
         }
+
+        internal static bool TryGetAccessorSymbolForFieldKeywordBinder(this Symbol symbol, out SourcePropertyAccessorSymbol accessorSymbol)
```

Comment:
>TryGetAccessorSymbolForFieldKeywordBinder [](http://example.com/codeflow?start=29&length=41)

I think the name is somewhat misleading. We are not doing any traversal or search, simply checking if the symbol satisfies some constraints. Perhaps rename to "NeedFieldKeywordBinder" or something similar and get rid of the out parameter.

Reply:
@AlekseyTs If removed the `out` parameter, then all callers will then have to cast the passed symbol if the condition is true. Is that okay? (number of callers isn't too small).

> NeedFieldKeywordBinder

This name *might* feel strange for this callsite:

https://github.com/dotnet/roslyn/blob/a38bf6d3e1df354ba1f82398eaef783da89f60ab/src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

It might be just me, so I'll rename if you think it's suitable for this callsite :)

Reply:
> If removed the out parameter, then all callers will then have to cast the passed symbol if the condition is true. Is that okay? 

That should be fine. A cast isn't that expensive, I think.

Reply:
Yeah I wasn't talking about it being expensive, just duplicate code I was trying to eliminate. But it's not that important. I'll get rid of the `out` parameter.

Reply:
> This name might feel strange for this callsite

The name was just an example, feel free to come up with a different name.